src/LibFTD2XX.jl

```diff
@@ -200,19 +200,30 @@ function status(handle::FT_HANDLE)
   mflaglist, lflaglist
 end
 
-function Base.nb_available(handle::FT_HANDLE)
-  nbrx = Ref{DWORD}()
-  status = ccall(cfunc[:FT_GetQueueStatus], cdecl, FT_STATUS, 
-                 (FT_HANDLE, Ref{DWORD}),
-                  handle,    nbrx)
-  FT_STATUS_ENUM(status) == FT_OK || throw(FT_STATUS_ENUM(status))
-  nbrx[]
+if VERSION < v"0.7"
+  function Base.nb_available(handle::FT_HANDLE)
+    nbrx = Ref{DWORD}()
+    status = ccall(cfunc[:FT_GetQueueStatus], cdecl, FT_STATUS, 
+                   (FT_HANDLE, Ref{DWORD}),
+                    handle,    nbrx)
+    FT_STATUS_ENUM(status) == FT_OK || throw(FT_STATUS_ENUM(status))
+    nbrx[]
+  end
+else
+  function Base.bytesavailable(handle::FT_HANDLE)
+    nbrx = Ref{DWORD}()
+    status = ccall(cfunc[:FT_GetQueueStatus], cdecl, FT_STATUS, 
+                   (FT_HANDLE, Ref{DWORD}),
+                    handle,    nbrx)
+    FT_STATUS_ENUM(status) == FT_OK || throw(FT_STATUS_ENUM(status))
+    nbrx[]
+  end
 end
 
-Base.eof(handle::FT_HANDLE) = (nb_available(handle) == 0)
+@compat Base.eof(handle::FT_HANDLE) = (bytesavailable(handle) == 0)
 
 function Base.readavailable(handle::FT_HANDLE)
-  @compat b = Vector{UInt8}(undef, nb_available(handle))
+  @compat b = Vector{UInt8}(undef, bytesavailable(handle))
```

Comment:
Please write 

```b = @compat Vector{UInt8}(undef, bytesavailable(handle))```

Just so we know exactly what the transform is

Reply:
done